melosys-eessi-app/src/main/java/no/nav/melosys/eessi/identifisering/event/BucIdentifisertEvent.java

```diff
@@ -7,11 +7,11 @@
 public class BucIdentifisertEvent extends ApplicationEvent {
 
     private final String bucId;
-    private final String aktørId;
+    private final String ident;
```

Comment:
er `ident` ett nytt begrep vi skal bruke for `aktørid` eller er det bare forkortet for `identifisert` i dette tilfellet for identifisert person for buc?

Reply:
Ident er kanskje litt generelt ja, da det ofte også brukes om saksbehandler-ident (men ikke i melosys-eessi). Det er vel brukt litt om hverandre i melosys-api også.

Men `aktørID` er en NAV-spesifikk person-id som ikke er det samme som et fnr/dnr. Men er fortsatt koblet til én person. Det er en generell anbefaling i NAV om at man går vekk fra å lagre `aktørID`, og heller lagrer fnr/dnr.

Kunne uansett brukt samme benevnelse som PDL har tatt i bruk, `folkeregisterident`, den er nok mer spesifikk.